### PR TITLE
data: GitWiz01

### PR DESCRIPTION
**Pull request by GitWiz01**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4857"></a>

### CODE REVIEW

File: data/GitWiz01.json
Comment (Panquesito7) [inline data/GitWiz01.json:5]: The `avatar` field is no longer required as of a new update.
```suggestion
```

File: data/GitWiz01.json
Comment (Panquesito7) [inline data/GitWiz01.json:4]: Missing comma.
```suggestion
  "bio": "A Web Dev Enthusiast who love open source",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by GitWiz01** [comment]
@Panquesito7 please merge this pr

**Comment by Panquesito7** [comment]
> @Panquesito7 please merge this pr

We'll merge it once another maintainer approves it. 🙂

File: data/GitWiz01.json
Comment (SaraJaoude) [inline data/GitWiz01.json:24]: You have a twitter url here (which leads to an account that does not exist) but a GitHub icon.  This is not a blocker so I will merge your PR.  Should you wish to make changes please open a new Pull Request.  You may wish to refer to the Editing guide in our Docs: https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**
